macos/FluentUIResources-macos/FluentUI-macos.xcassets/ButtonColors/neutralBackground2.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0xFF",
+          "green" : "0xFF",
+          "red" : "0xFF"
+        }
+      },
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
```

Comment:
this should be hex rather than floating number which could be inconsistent.

Reply:
same comments for other color files

Reply:
Any way you know of for Xcode to default to hex values when I'm working with color assets?  Changing this drop-down for every single color is getting really old...